frontend/src/routes/manage/Video/TechnicalDetails.tsx

```diff
@@ -0,0 +1,162 @@
+import { useTranslation } from "react-i18next";
+
+import { NotAuthorized } from "../../../ui/error";
+import { useUser } from "../../../User";
+import { Breadcrumbs } from "../../../ui/Breadcrumbs";
+import { PageTitle } from "../../../layout/header/ui";
+import { AuthorizedEvent, makeManageVideoRoute, PAGE_WIDTH } from "./Shared";
+import { CopyableInput } from "../../../ui/Input";
+
+
+export const ManageVideoTechnicalDetailsRoute = makeManageVideoRoute(
+    "technical-details",
+    "/technical-details",
+    event => <Page event={event} />,
+);
+
+type Props = {
+    event: AuthorizedEvent;
+};
+
+const Page: React.FC<Props> = ({ event }) => {
+    const { t } = useTranslation();
+
+    const breadcrumbs = [
+        { label: t("manage.management"), link: "/~manage" },
+        { label: t("manage.my-videos.title"), link: "/~manage/videos" },
+        { label: event.title, link: `/~manage/videos/${event.id.substring(2)}` },
+    ];
+
+    const user = useUser();
+    if (user === "none" || user === "unknown") {
+        return <NotAuthorized />;
+    }
+
+    return <>
+        <Breadcrumbs path={breadcrumbs} tail={t("manage.my-videos.technical-details.title")} />
+        <PageTitle title={t("manage.my-videos.technical-details.title")} />
+        <div css={{
+            maxWidth: PAGE_WIDTH,
+            "& > section:not(:last-child)": {
+                marginBottom: 32,
+            },
+            "& > section > h2": {
+                fontSize: 18,
+                marginBottom: 4,
+            },
+        }}>
+            <OpencastId event={event} />
+            <TrackInfo event={event} />
+            <OtherInfo event={event} />
+        </div>
+    </>;
+};
+
+const OpencastId: React.FC<Props> = ({ event }) => {
+    const { t } = useTranslation();
+
+    return <section>
+        <h2>{t("manage.my-videos.technical-details.opencast-id")}</h2>
+        <CopyableInput
+            value={event.opencastId}
+            css={{ width: 400, fontFamily: "monospace", fontSize: 14 }}
+        />
+    </section>;
+};
+
+const TrackInfo: React.FC<Props> = ({ event }) => {
+    const { t } = useTranslation();
+
+    if (event.syncedData == null) {
+        return null;
+    }
+
+    type SingleTrackInfo = {
+        resolution: readonly number[] | null;
+        mimetype: string | null;
+    };
+
+    const flavors: Map<string, SingleTrackInfo[]> = new Map();
+    for (const { flavor, resolution, mimetype } of event.syncedData.tracks) {
+        let tracks = flavors.get(flavor);
+        if (tracks === undefined) {
+            tracks = [];
+            flavors.set(flavor, tracks);
+        }
+
+        tracks.push({ resolution, mimetype });
+    }
+
+
+
+    return <section>
+        <h2>{t("manage.my-videos.technical-details.tracks")}</h2>
+        <ul css={{ fontSize: 15, marginTop: 8, paddingLeft: 24 }}>
+            {Array.from(flavors, ([flavor, tracks]) => <li key={flavor}>
```

Comment:
I feel like `.entries().map(...)` would be more idiomatic 🤔

Reply:
That's what I wrote first. But `entries` returns an iterable that does not have the `map` method. Wöp wöp.

Reply:
OH! :eyes: